study.md

```diff
@@ -13,6 +13,10 @@ please do the following and be prepared to share and discuss during our next
 class.
 
 Submit detailed answers to these in this file via a pull request.
+You may want to submit pictures for your wireframes and/or user stories.
+[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+has instructions to link to a picture you've uploaded to a service like [Imgur]
+(http://imgur.com/).
```

Comment:
Would it be good to include the GitHub Issues trick as an alternative? http://solutionoptimist.com/2013/12/28/awesome-github-tricks/